and-bible/app/src/main/res/xml/settings.xml

```diff
@@ -81,6 +81,12 @@
 				android:title="@string/prefs_open_links_in_special_window_title"
 				android:summary="@string/prefs_open_links_in_special_window_summary"
 				android:defaultValue="true"/>
+	<ListPreference android:key="split_mode_pref"
+				android:title="Split mode"
```

Comment:
These 2 strings should be moved to strings.xml to enable translation

Reply:
I can fix this, but please note that other strings (titles and summaies) in settings.xml are not translated either. Or did I miss something?

Reply:
Fixed.

Reply:
They are all translated, it is just Intellij Studio magic that shows the actual string instead of the resource id.